test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"k8s.io/kubernetes/pkg/apis/flowcontrol"
```

Comment:
admission test should work for generic types, referencing a specific group is suspisicious

Reply:
I suppose that is to support this:
```
	prioLevelConfRef = flowcontrol.PriorityLevelConfigurationReference{
		Name: "system-bar",
	}
```
but I do not know what that is there for.  I do not understand this test.

Reply:
once you add the stub data in the etcd integration test data file, you shouldn't need to touch this file at all